administrator-guides/federation/README.md

```diff
@@ -1,11 +1,88 @@
 # Federation
 
+## Introduction
+
+Federation allows communication with other Rocket.Chat servers.
+
+When you open Federation's config screen, you will see this:
+
+![Initial Federation Screen](./initial.png){:style="width='100%' height='auto'"}
+
+Below, a brief explanation of each configuration option:
+
+#### Enabled
+
+This determines whether or not Federation is enabled on this server.
+
+#### Status
+
+This is the current Federation status, which can be one of the following:
+
+- **Could not enable, settings are not fully set :** Make sure all the options are correctly filled and saved;
+- **Booting... :** federation is initializing;
+- _[HUB Only]_ **Registering with Hub... :** federation is trying to register with the Hub;
+- _[HUB Only]_ **Disabled, could not register with Hub :** federation could not register with the Hub;
+- **Disabled :** federation is disabled
+- **Running :** federation is running and ready to communicate with other servers;
+
+#### Domain
+
+Fill this option with your domain name, usually where the Rocket.Chat server is hosted.
+
+#### Public Key
+
+All Federation communications are double encrypted, which means messages cannot be delivered by mistake to a wrong peer. This is your public key, which is specially generated for Rocket.Chat and by Rocket.Chat, which will be shared with peers you are communicating with.
+
+#### Hub URL
+
+This usually remains as is. This is the Hub URL, where your server will register and confirm domain ownership if you are not using DNS discovery method.
+
+#### Discovery Method
+
+How are you going to be found in the Rocket.Chat Federated Network? You can choose through DNS (recommended) or Rocket.Chat Hub.
+
+## Adding Federation to your server
+
 You have two ways of adding your server to the Rocket.Chat Federated Network:
 
-## Configuring your DNS
+### Recommended: Configuring your DNS
+
+To add your server to the Federated Network using the DNS, fill all the needed configuration fields (making sure you are picking **true** on the _Enabled_ field), pick the `DNS` _Discovery Method_ option, save and follow the guide below.
+
+Let's suppose we have a domain named `mydomain.com`, and my Rocket.Chat server is hosted at `chat.mydomain.com`, port `80`.
+
+You'll have to add two DNS records:
+
+#### SRV Record
+
+- Service: `_rocketchat`
+- Protocol: `_tcp`
+- Name: `mydomain.com`
+- Weight: `1`
+- Priority: `1`
+- TTL: `1 hour`
+- Target: `chat.mydomain.com`
+- Port: `80`
+
+#### TXT Record
+
+- Host: `rocketchat-public-key`
+- Value: `<my public key, as shown on the configuration screen>`
```

Comment:
should we add an special tip for Route53 users regarding need to split the key in two string?

Reply:
@sampaiodiego I thought about it, but there are so many different DNS managers out there, it is going to be hard to, eventually, add instructions for all of them.

I thought about saying that some DNS providers won't allow the full string to be entered, so after saving, we recommend making sure the string is complete and if it is not, the string needs to be entered in more than one key, what do you think?

Reply:
I like it, I think it is enough saying that way

Reply:
This should be made more clear as it reads now... I would want to create a dns record at rocketchat-public-key.mydomain.com

Reply:
@geekgonecrazy is TXT a single record?

Reply:
For route 53 yeah its one big ugly record :grin: 

Its not the end of the world.. but I know for DKIM most of the time they have some dedicated TXT record just for it and nothing else

Reply:
Yeah, I just saw that 🤦‍♂️ 

We could change that, should be easy. @sampaiodiego what do you think?

Reply:
@geekgonecrazy we are going to make that change, thanks for bringing that